src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -1875,6 +1875,41 @@ public void TestLongProcessIsWorking()
             Assert.True(p.HasExited);
         }
 
+       [Fact]
```

Comment:
Nit: indent :)

Reply:
at the end of the day my eyes are fried 😄 thank's